test/de.cau.cs.kieler.klighd.piccolo.test/src/de/cau/cs/kieler/klighd/piccolo/test/highlightedEdgeToForeground/HighlightedEdgeToForegroundTest.java

```diff
@@ -89,58 +89,64 @@ public void configure(final Binder binder) {
-//        for (int i = 0; i < 3; i++) {
-//            clickOn(4, 2);
-//            waitAmoment();
-//            clickOn(4, 4);
-//            waitAmoment();
-//        }
-//        clickOn(4, 4);
-//        waitAmoment();
-//
-//        // The following is display a "warm up" test since the continuous build's (virtual) display's palette
-//        //  seems to be very limited. The observed behavior is: If 'black' or 'white' is the first observed
-//        //  color, only those colors will be obtained subsequently. I interpret this as follows:
-//        // The image's color palette then contains only black and white. If the image also contains 'red'
-//        //  parts before testing the first time for any color the palette than also contains 'red'.
-//        // Weird ...
-//        final KShapeLayout firstChildNodeLayout = viewContext.getViewModel().getChildren().get(0).getChildren().get(0);
-//        final int firstChildNodeYPos = Math.round(firstChildNodeLayout.getYpos());
-//        final int firstClickXPos = 5 + 100;
-//
-//        clickOn(firstClickXPos, firstChildNodeYPos);
-//        waitAmoment();
-//
-//        clickOn(4, 4);
-//        waitAmoment();
+        //
+        // // The following might be helpful when running/debugging via Eclipse:
+        //
+        // // The following is some mouse pointer "warm up" since I experienced some delayed
+        // reactions of the
+        // // UI on the mouse pointer movement commands executed in the tests below.
+        // // Note, that these movements are not mandatory for the test execution,
+        // // but very helpful for developers for verifying the test execution.
+        // for (int i = 0; i < 3; i++) {
+        // clickOn(4, 2);
+        // waitAmoment();
+        // clickOn(4, 4);
+        // waitAmoment();
+        // }
+        // clickOn(4, 4);
+        // waitAmoment();
+        //
+        // // The following is display a "warm up" test since the continuous build's (virtual)
+        // display's palette
+        // // seems to be very limited. The observed behavior is: If 'black' or 'white' is the first
+        // observed
+        // // color, only those colors will be obtained subsequently. I interpret this as follows:
+        // // The image's color palette then contains only black and white. If the image also
+        // contains 'red'
+        // // parts before testing the first time for any color the palette than also contains
+        // 'red'.
+        // // Weird ...
+        // final KShapeLayout firstChildNodeLayout =
+        // viewContext.getViewModel().getChildren().get(0).getChildren().get(0);
+        // final int firstChildNodeYPos = Math.round(firstChildNodeLayout.getYpos());
+        // final int firstClickXPos = 5 + 100;
+        //
+        // clickOn(firstClickXPos, firstChildNodeYPos);
+        // waitAmoment();
+        //
+        // clickOn(4, 4);
+        // waitAmoment();
```

Comment:
This change seems unnecessary, especially as it destroys the indentation in the commented out code